Sources/SwiftInspectorVisitors/PropertyVisitor.swift

```diff
@@ -41,7 +41,7 @@ public final class PropertySyntaxVisitor: SyntaxVisitor {
         // e.g. let red: Int, green, blue: Double
         // where both green and blue are of type Double
         if let typeName = typeName { lastFoundType = typeName }
-        propertiesInfo.insert(.init(
+        properties.append(.init(
                                 name: identifier.identifier.text,
```

Comment:
can we re-indent post name-change? here and elsewhere

Reply:
I really need to add linting for this repo 😁